dali/test/python/test_operator_to_decibels.py

```diff
@@ -94,3 +94,60 @@ def test_operator_to_decibels_vs_python():
                                                             (20.0, 1e-6, -120.0, (2, 3, 40))]:
                 yield check_operator_to_decibels_vs_python, device, batch_size, shape, \
                     multiplier, reference, cutoff_db
+
+
+class NaturalLogarithmPipeline(Pipeline):
```

Comment:
You can add a test for this enforce as well.

Reply:
Done